doc improvements: vsc: extend include for building steps (#7767)
nrfconnect/sdk-nrf: doc/nrf/ug_zigbee_qsg.rst

For this quick start guide, we will install the following software:

* Toolchain Manager - An application for installing the full |NCS| toolchain.

> * |VSC| (VS Code) - The recommended IDE for the |NCS|.

Reviewer: ```suggestion
* Microsoft's Visual Studio Code - The recommended IDE for the |NCS|.
```

This is actually not the correct shortcut. |VSC| is actually a shortcut for "nRF Connect for Visual Studio Code".

> * nRF Connect for |VSC| - Extension for VS Code that allows you to develop applications for the |NCS|.

Reviewer: ```suggestion
* |VSC| - Extension for VS Code that allows you to develop applications for the |NCS|.
```
|VSC| is actually a shortcut for "nRF Connect for Visual Studio Code".

#. Click the :guilabel:`Open VS Code` button.
   The installation wizard checks whether you have the following software installed:

>    * |VSC|

Reviewer: ```suggestion
   * Microsoft's Visual Studio Code
```

|VSC| is actually a shortcut for "nRF Connect for Visual Studio Code".

>    * nRF Connect for |VSC| extension

Reviewer: ```suggestion
   * |VSC|
```

|VSC| is actually a shortcut for "nRF Connect for Visual Studio Code".
